wgpu-types/src/lib.rs

```diff
@@ -173,6 +173,12 @@ bitflags::bitflags! {
     #[repr(transparent)]
     #[derive(Default)]
     pub struct Features: u64 {
+        //
+        // ---- Start numbering at 1 << 0, each one defined in terms of the last ----
```

Comment:
This is an interesting idea. The idea is to let us insert new values wherever we like, without having to renumber everything, right?

I just hope we don't find ourselves needing to understand hex `Features` values, because it'll be a bear to figure out anything's actual numeric value. (Note that this has no `Debug` impl.)

Thank you for the new docs!

Reply:
All bitflags have debug impls, we use them all over for printing features.

But yes, this way we're just inserting without needing to do that error prone and menial work